server/chat-commands.js

```diff
@@ -671,24 +671,29 @@ const commands = {
 	awayhelp: [`/away - Marks you as away. Send a message or use /back to indicate you are back.`],
 
 	'!back': true,
+	clearstatus: 'back',
 	unaway: 'back',
 	unafk: 'back',
 	back(target, room, user) {
-		// This suffices for now, but will need to be updated
-		// if we ever expand upon the busy status.
-		if (user.status.startsWith('(Busy)')) {
+		if (!user.status) return;
+		const statusType = user.isAway() ? 'away' : user.status.startsWith('(Busy)') ? 'busy' : null;
+		user.clearStatus();
+
+		if (statusType === 'busy') {
```

Comment:
Why would we not also want to block these for 'Away', given not doing so kind of breaks the past behavior of the `/away` command?